indico/modules/rb_new/client/styles/palette.scss

```diff
@@ -22,12 +22,10 @@ $highlight-color: #7dcfb6;
 $dark-highlight-color: #45ba97;
 $primary-color: #2185d0;
 
-$sui-red: #db2828;
-$sui-orange: #f2711c;
-$sui-green: #21ba45;
-$sui-grey: #767676;
-
-$light-red: #ffbdbd;
+$sui-red: $red;
+$sui-orange: $orange;
+$sui-green: $green;
+$sui-grey: $gray;
```

Comment:
:see_no_evil: yay for different spellings of gr[ea]y :disappointed: 

`s/sui-grey/sui-gray/` after merging this?

Reply:
Actually, `$sui-*` will go away, now that the palettes are unified.

Reply:
... done.